orchagent/bufferorch.cpp

```diff
@@ -784,6 +787,32 @@ task_process_status BufferOrch::processQueue(KeyOpFieldsValuesTuple &tuple)
                     return handle_status;
                 }
             }
+
+            /* when we apply buffer configuration we need to increase the ref counter of this port
+             * or decrease the ref counter for this port when we remove buffer cfg
+             * so for each priority cfg in each port we will increase/decrease the ref counter
+             * also we need to know when the set command is for creating a buffer cfg or modifying buffer cfg -
+             * we need to increase ref counter only on create flow.
+             * so we added a map that will help us to know what was the last command for this port and priority -
+             * if the last command was set command then it is a modify command and we dont need to increase the buffer counter
+             * all other cases (no last command exist or del command was the last command) it means that we need to increase the ref counter */
+            if (op == SET_COMMAND) {
+                if (queue_port_flags[port_name][ind] != SET_COMMAND) {
```

Comment:
Do you see any issues when first time the `queue_port_flags[port_name][ind]` was not set yet?

Reply:
I dont think we have an issue on the first time sine